crates/opencascade-sys/build.rs

```diff
@@ -33,10 +38,18 @@ fn main() {
     println!("cargo:rustc-link-lib=static=TKBO");
     println!("cargo:rustc-link-lib=static=TKOffset");
 
+    let mut occ_convert_define = None;
+    if is_windows {
+        println!("cargo:rustc-link-lib=dylib=user32");
+        occ_convert_define = Some("FALSE");
+    }
+
     cxx_build::bridge("src/lib.rs")
         .cpp(true)
         .flag_if_supported("-std=c++11")
-        .include(format!("{}", dst.join("include").join("opencascade").display()))
+        .define("_USE_MATH_DEFINES", "TRUE")
+        .define("OCC_CONVERT_SIGNALS", occ_convert_define)
```

Comment:
From [this page](https://dev.opencascade.org/doc/refman/html/_standard___error_handler_8hxx.html)

> On Windows with MSVC compiler, exceptions can be thrown directly from signal handler, this OCC_CONVERT_SIGNALS is not needed. Note however that this requires that compiler option /EHa is used.

Not sure if we need `/EHa` here or not. I built this on the default MSVC toolchain you get with a windows Rust installation and I get two linking errors:

```
unresolved external symbol "public: void __cdec1 Standard_ErrorHandler::Callback::RegisterCallback(void)"
unresolved external symbol "public: void __cdec1 Standard_ErrorHandler::Callback::UnregisterCallback(void)"
```

Currently still working this out.

Reply:
Ive limited the flag to only `*-windows-gnu` builds. It should fix the error for you now.

Reply:
@DSchroer I think if it's not `*-windows-gnu` you don't want to define `OCC_CONVERT_SIGNALS` at all. On MSVC, if you set `occ_convert_define` to `None`, you're still defining `OCC_CONVERT_SIGNALS` and I still get the issue I mentioned above.

Reply:
Oh shoot. I assumed based on the interface `.define("thing", None)` would disable it. Ill look into it soon.

Reply:
Ive done another pass. This time its only ever set for `windows-gnu`. Hopefully this works.